File: 250721/윤년인가/is-leap-year.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intY = sc.nextInt();

        // 변수 정의
        String strResult;
        // 조건문
        if (intY % 4 == 0) {
            if (intY % 100 == 0 && intY % 400 != 0) {
                strResult = "false";
            } else {
                strResult = "true";
            }
        } else {
            strResult = "false";
        }

        // 출력
        System.out.println(strResult);
    }
}

File: 250721/윤년인가/is-leap-year.py
# 입력
intY = int(input())

# 조건문
if intY % 4 == 0 :
    if intY % 100 == 0 and intY % 400 != 0 :
        print("false")
    else :
        print("true")
else :
    print("false")